app/assets/stylesheets/pictures.scss

```diff
@@ -0,0 +1,37 @@
+// Place all the styles related to the pictures controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
+article{
+  background: white;
+  padding: 0;
+  display: block;
+  margin: 70px 0;
+  border: 1px solid #edeeee;
+  border-radius: 3px;
+  header{
+    height: 64px;
+    padding: 14px 20px;
+    a{
+      display: inline-block;
+      padding: 10px 0;
+    }
+  }
+  figure{
+    display: block;
+    height: 500px;
+    border-top: 1px solid #edeeee;
+    border-bottom: 1px solid #edeeee;
+    background: center center no-repeat;
+    background-size: contain;
+  }
+  .details{
+    padding: 24px;
+  }
+}
+
+article.no-pictures{
```

Comment:
Merge rule `article.no-pictures` with rule on line 4<br>Avoid qualifying class selectors with an element.<br>Opening curly brace `{` should be preceded by one space